events/models.py

```diff
@@ -518,6 +535,20 @@ def get_enrolment_count(self):
         except AttributeError:
             return self.enrolments.count()
```

Comment:
As the annotated value uses distinct, should distinct be added here as well?

Reply:
After discussion this should be fine, there shouldn't be need for distinct here because "enrolments" is the related_name in the ForeignKey references from Enrolment to Occurence.